docs/DevelopmentSetup/windows_installation.rst

```diff
@@ -29,9 +29,19 @@ Start a command shell in the Vcpkg repository folder (that you had cloned earlie
     # install packages
     vcpkg.exe install boost-test:x64-windows boost-program-options:x64-windows boost-log:x64-windows turtle:x64-windows zipper:x64-windows poco:x64-windows libpq:x64-windows gdal:x64-windows sqlite3:x64-windows boost-ublas:x64-windows
 
+.. figure:: ../images/installation_vs2019_flint.example/Step1b.png
+  :width: 600
+  :align: center
+  :alt: Alternative text
+
+  Installing required packages using vcpkg in Command Prompt
+
 Using cmake to build the project
 --------------------------------
 
+.. note::
+   Please make sure that you have checked out to the ``Develop`` Branch for the FLINT Installation.
+   
```

Comment:
Could add the commands for first timers:

```
git checkout develop
git pull
```

Reply:
Updated